tests/resources/app/config/CacheBox.cfc

```diff
@@ -0,0 +1,50 @@
+﻿component{
```

Comment:
Sure. I was just following Eric's conventions for his template. I'll submit a PR to him as well.